### PR TITLE
WIP Add support for INFO os_min_ver

### LINKED ISSUE #19: Invalid "startable" attribute in spkrepo since INFO change
After INFO generation has been cleaned up from default values https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.spk.mk#L165,
spkrepo reports improperly "startable" attribute

### PR DESCRIPTION
**Pull request by ymartin59**
Deprecated firmware property is now optional
Fixes #19

### CODE REVIEW

**Comment by Safihre** [comment]
👍

**Comment by ymartin59** [comment]
Just deployed in production for "live testing"... seems like OK

**Comment by ymartin59** [comment]
`startable` flag is still not correct, now always true, even if INFO contains:
```
startable="no"
ctl_stop="no"
```

File: spkrepo/views/api.py [hunk 1]
```diff
@@ -139,7 +142,7 @@ def post(self):
                               conf_privilege=spk.conf_privilege,
                               conf_resource=spk.conf_resource,
                               install_wizard='install' in spk.wizards, upgrade_wizard='upgrade' in spk.wizards,
-                              startable=spk.info.get('startable'), license=spk.license)
+                              startable=spk.info.get('startable') or True, license=spk.license)
```
Comment (Safihre) [inline spkrepo/views/api.py]: Maybe extract the `ctl_stop` and use it here?

**Comment by Safihre** [comment]
:+1:

**Pull request merged**